src/lib/secretsManager.ts

```diff
@@ -64,4 +69,4 @@ const remove = async (secretId: string) => {
   }
 }
 
-export default { getAll, get, upsert, remove }
+export default { getAll, getAllByUser, get, upsert, remove }
```

Comment:
is getAll still needed?

Reply:
yes, when saving a new secret we get secrets for all users and push the new one before saving all of them in storage.

Reply:
but does it need to be exported?

Reply:
Right. We don't need to export it